glambie/data/timeseries_data.py

```diff
@@ -0,0 +1,72 @@
+
+from decimal import Decimal, getcontext
+import numpy as np
+from typing import Optional
+import pandas as pd
+
+
+class ChangeTimeseries():
+    """Class containing a data series and corresponding metadata
+    """
+    @property
+    def min_time(self) -> float:
+        return self._get_min_time()
```

Comment:
As this is a property, there's no need to put the logic that evaluates the property in a separate function. As a result, there are two redundant functions in this class.

Reply:
done